public/mentor/index.html

```diff
@@ -0,0 +1,340 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>創業者共創平台</title>
+    <link rel="icon" href="../_assets/images/favicon.svg" />
+    <link rel="stylesheet" href="../_assets/styles/main.css" />
+  </head>
+  <body>
```

Comment:
In an HTML document, **ONLY ONE** `<body></body>` element can exist.

- [`<body>`: The Document Body element | MDN Web Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/body)